xql/README.md

```diff
@@ -10,8 +10,9 @@ Running SQL like queries on Xarray Datasets. Consider dataset as a table and dat
 * **`group by` Functions** - This is supported on the coordinates only. e.g. time, latitude, longitude, etc.
 * **`aggregate` Functions** - Aggregate functions `AVG()`, `MIN()`, `MAX()`, etc. Only supported on data variables.
 * For more checkout the [road-map](https://github.com/google/weather-tools/tree/xql-init/xql#roadmap).
-> Note: For now, we support `where` conditions on coordinates only.
-> Note: For now, Only a single aggregate function is supported per query.
+> Note: For now, we support `where` conditions on coordinates only. 
+
+> Note: For now, variable with starting from the numbers(0-9) in aggregation function is not supported.
```

Comment:
Note: Applying aggregate functions on variables starting with numbers(0-9) are not supported. e.g. AVG(100m_u_component_of_wind).